ztp/siteconfig-generator/siteConfig/siteConfigBuilder.go

```diff
@@ -78,20 +73,23 @@ func (scbuilder *SiteConfigBuilder) getClusterCRs(clusterId int, siteConfigTemp
 			cluster := siteConfigTemp.Spec.Clusters[clusterId]
 			dataMap, err := scbuilder.getExtraManifest(dataMap, cluster)
 			if err != nil {
-				return clusterCRs, err
+				log.Printf("warning could not read extra-manifest %s.%s %s\n", cluster.ClusterName, cluster.ExtraManifestPath, err)
 			}
 
-			// FIXME: Assuming 1 node for SNO deployment needs to be changed for RWN deployment
-			if len(siteConfigTemp.Spec.Clusters[clusterId].Nodes) > 0 {
+			// Adding workload partitions MC only for SNO clusters.
```

Comment:
Nice!

Reply:
Agreed. Really nice feature to have but would prefer separate PRs for different features. It makes tracking, backporting, and  reverting a feature (when necessary) easier.